apps/sparo-lib/src/cli/commands/fetch.ts

```diff
@@ -46,7 +41,7 @@ export class FetchCommand implements ICommand<IFetchCommandOptions> {
       fetchArgs.push('origin', branch);
```

Comment:
Should this be `_getBranchRemote()` instead of `origin`?

Reply:
Nice catch!